test/profiling/scenario2.js

```diff
@@ -16,8 +16,8 @@ let myChat = null;
 let chats = [];
 
 let ChatEngine = ChatEngineCore.create({
-    publishKey: 'pub-c-dc3a2375-67aa-400f-b174-1382afae525a',
-    subscribeKey: 'sub-c-6517388c-88d4-11e7-a576-2a15881e5abd',
+    publishKey: process.env.PUB_KEY_2,
```

Comment:
Can you put these all on one set of keys? I believe the only reason these are two sets of keys is because someone else wrote the profiling code.

Reply:
Aww for real? Yea, lemme see about cleaning that up then.